EIPS/eip-7702.md

```diff
@@ -1,19 +1,19 @@
 ---
 eip: 7702
-title: Set EOA account code for one transaction
-description: Add a new tx type that sets the code for an EOA during one transaction execution
+title: Set EOA account code
+description: Add a new tx type that sets the code for an EOA during execution
 author: Vitalik Buterin (@vbuterin), Sam Wilson (@SamWilsn), Ansgar Dietrichs (@adietrichs), Matt Garnett (@lightclient)
 discussions-to: https://ethereum-magicians.org/t/eip-set-eoa-account-code-for-one-transaction/19923
 status: Review
 type: Standards Track
 category: Core
 created: 2024-05-07
-requires: 2718, 2929, 2930
+requires: 2718, 2929, 2930, 3541, 3607
 ---
 
 ## Abstract
 
-Add a new transaction type that adds a list of `[address, y_parity, r, s]` authorization tuples, and converts the signing accounts (not necessarily the same as the `tx.origin`) into smart contract wallets for the duration of that transaction.
+Add a new transaction type that adds a list of `[chain_id, address, nonce, y_parity, r, s]` authorization tuples. For each tuple, write a delegation designator `(0xef0000 ++ address)` to the signing account's code. All code reading operations must load the pointed to by the designator.
```

Comment:
Minor: last sentence is missing a noun (must load the ??? pointed to)

Reply:
fixed thx